data-raw/healthy-lives/unemployment.R

```diff
@@ -0,0 +1,48 @@
+# ---- Load packages ----
+library(httr)
+library(readxl)
+library(tidyverse)
+library(geographr)
+
+# ---- Get data ----
+# Source: https://www.ons.gov.uk/employmentandlabourmarket/peoplenotinwork/unemployment/datasets/modelledunemploymentforlocalandunitaryauthoritiesm01
+GET(
+  "https://www.ons.gov.uk/file?uri=/employmentandlabourmarket/peoplenotinwork/unemployment/datasets/modelledunemploymentforlocalandunitaryauthoritiesm01/current/modelbasedunemploymentdataaugust2022.xls",
+  write_disk(tf <- tempfile(fileext = ".xls"))
+)
+
+unemployment_raw <-
+  read_excel(tf, sheet = 4, skip = 2)
```

Comment:
You can specify the range of cells to read in so you don't need to go through 200+ columns